spec/block_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "spec_helper"
+require "Date"
+
+describe "Block class" do
+  before do
+    @rooms = []
+    20.times do |k|
+      @rooms << Hotel::Room.new(id: k)
+    end
+    @hotel = Hotel::Hotel.new(
+      id: 1,
+      rooms: @rooms,
```

Comment:
Rather than messing around with an entire `Hotel` in the `Block` tests, you should instantiate a `Block` directly and test on that.